tests/scripts/thread-cert/Cert_9_2_04_ActiveDataset.py

```diff
@@ -112,7 +118,7 @@ def test(self):
             extended_panid='000db70000000000',
             master_key='00112233445566778899aabbccddeeff',
             mesh_local='fd00:0db7::',
-            network_name='UL',
+            network_name='GRL',
```

Comment:
Can the original code work since the `network_name` is wrong?

Reply:
align with test plan, this dataset is rejected